easybuild/easyblocks/m/mxnet.py

```diff
@@ -169,7 +187,7 @@ def extensions_step(self):
         # MXNet doesn't provide a list of its R dependencies by default
         write_file("NAMESPACE", R_NAMESPACE)
         change_dir(self.mxnet_src_dir)
-        self.r_ext.prerun()
+        self.r_ext.prerun()  # tried commenting this out
```

Comment:
So, should we comment this out or not? why?

Reply:
no, sorry just a leftover debug thing

Reply:
solved